src/Symfony/Component/BrowserKit/Client.php

```diff
@@ -563,6 +574,23 @@ public function followRedirect()
         return $response;
     }
 
+    /**
+     * Determines if the response has a meta refresh.
+     *
+     * @return string|false Redirect URI if response has a meta refresh
+     */
+    protected function hasMetaRefresh()
```

Comment:
this method should be private and should return `string|null` (thus, the docblock could/should be removed, and a return type added: `?string` instead.)
The name should be getMetaRefresh as one would expect `has*` methods to return a bool IMHO.